OracleDatabase/SingleInstance/dockerfiles/19.3.0/createDB.sh

```diff
@@ -87,11 +87,18 @@ echo "$ORACLE_PDB=
   )" >> $ORACLE_HOME/network/admin/tnsnames.ora
 
 # Remove second control file, fix local_listener, make PDB auto open, enable EM global port
+# Create externally mapped oracle user for health check
 sqlplus / as sysdba << EOF
    ALTER SYSTEM SET control_files='$ORACLE_BASE/oradata/$ORACLE_SID/control01.ctl' scope=spfile;
    ALTER SYSTEM SET local_listener='';
    ALTER PLUGGABLE DATABASE $ORACLE_PDB SAVE STATE;
    EXEC DBMS_XDB_CONFIG.SETGLOBALPORTENABLED (TRUE);
+
+   alter session set "_oracle_script" = true;
+   CREATE USER OPS\$oracle IDENTIFIED EXTERNALLY;
+   grant create session, select any dictionary to OPS\$oracle;
```

Comment:
The select grant should only be given to the one view if the user is only used for that one view.

Reply:
Done, I limited grants, and hope it is enough to just check `v$pdbs`